testssl.sh

```diff
@@ -18597,7 +18600,7 @@ mybanner() {
      bb1=$(cat <<EOF
 
 ###########################################################
-    $PROG_NAME       $VERSION from 
+    $PROG_NAME       $VERSION from
```

Comment:
This blank is needed

Reply:
I did not delete those, Atom just removes stuff I guess.

Reply:
You're not the first. :-)  I think I need a workaround for atom and vim users who automatically squash those blanks.